test_app/tests/authentication/serializers/test_authententicator_map.py

```diff
@@ -0,0 +1,126 @@
+from unittest.mock import MagicMock
+
+import pytest
+from rest_framework.serializers import ValidationError
+
+from ansible_base.authentication.serializers.authenticator_map import AuthenticatorMapSerializer
+from test_app.tests.authentication.conftest import ORG_MEMBER_ROLE_NAME, SYSTEM_ROLE_NAME, TEAM_MEMBER_ROLE_NAME
+
+
+@pytest.fixture
+def serializer():
+    return AuthenticatorMapSerializer()
+
+
+class TestAuthenticatorMapSerializerMapType:
+    @pytest.fixture(autouse=True)
+    def init_serializer(self, serializer):
+        serializer.validate_trigger_data = MagicMock(return_value={})
+        serializer.validate_role_data = MagicMock(return_value={})
+
+    def test_validate_map_type_organization(self, serializer):
+        """map_type='organization' requires fields organization and role"""
+        with pytest.raises(ValidationError) as e:
+            serializer.validate(dict(name="authentication_map_1", map_type="organization"))
+        assert set(e.value.detail.keys()) == {'organization', 'role'}
+
+        with pytest.raises(ValidationError) as e:
+            serializer.validate(dict(name="authentication_map_1", map_type="organization", organization='test_org'))
+        assert list(e.value.detail.keys()) == ['role']
+
+        serializer.validate(dict(name="authentication_map_1", map_type="organization", organization='test_org', role=SYSTEM_ROLE_NAME))
```

Comment:
You didn't include the `system_role` fixture, so using `role=SYSTEM_ROLE_NAME` should throw a `ValidationError` because the role doesn't exist.

Am I missing something?